Gemfile

```
@@ -19,10 +20,14 @@ group :development, :test do
 end
 
 group :development do
-  gem "faker", "~> 2.14"
   gem "letter_opener_web", "~> 2.0"
   gem "listen", "~> 3.1"
+  gem "rubocop-faker"
```

Can we add a pin for this version? 

```suggestion
  gem "rubocop-faker", "~> 1.1"
```

(This is done automatically if you use `bundle add rubocop-faker` for instance)

---

done